src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -1671,6 +1671,61 @@ public void fieldCollapsingTest() throws IOException {
         Assert.assertEquals(21, hits.length());
     }
 
+    @Test
+    public void backticksQuotedIndexNameTest() throws Exception {
+        AdminClient adminClient = this.admin();
+        TestUtils.createTestIndex(adminClient, "bank_unquote", "bank_unquote", null);
+        TestUtils.loadBulk(ESIntegTestCase.client(),
+                "/src/test/resources/bank_for_unquote_test.json", "bank");
+
+        String query = "SELECT lastname FROM `bank` ORDER BY age LIMIT 3";
+        JSONArray hits = getHits(executeQuery(query));
+
+        assertThat("bank", equalTo(((JSONObject) hits.get(0)).query("/_index")));
+    }
+
+    @Test
+    public void backticksQuotedFieldNamesTest() {
+        String expected = executeQuery(StringUtils.format("SELECT b.lastname FROM %s " +
+                "AS b ORDER BY age LIMIT 3", TestsConstants.TEST_INDEX_BANK), "jdbc");
+        String quotedFieldResult = executeQuery(StringUtils.format("SELECT b.`lastname` FROM %s " +
+                "AS b ORDER BY age LIMIT 3", TestsConstants.TEST_INDEX_BANK), "jdbc");
+
+        assertEquals(expected, quotedFieldResult);
+    }
+
+    @Test
+    public void backticksQuotedAliasTest() {
+        String expected = executeQuery(StringUtils.format("SELECT b.lastname FROM %s " +
+                "AS b ORDER BY age LIMIT 3", TestsConstants.TEST_INDEX_BANK), "jdbc");
+        String quotedAliasResult = executeQuery(StringUtils.format("SELECT `b`.lastname FROM %s" +
+                " AS `b` ORDER BY age LIMIT 3", TestsConstants.TEST_INDEX_BANK), "jdbc");
+        String quotedAliasAndFieldResult = executeQuery(StringUtils.format("SELECT `b`.`lastname` FROM %s " +
+                "AS `b` ORDER BY age LIMIT 3", TestsConstants.TEST_INDEX_BANK), "jdbc");
+
+        assertEquals(expected, quotedAliasResult);
+        assertEquals(expected, quotedAliasAndFieldResult);
+    }
+
+    @Test
+    public void backticksQuotedAliasWithSpecialCharactersTest() {
```

Comment:
Are there any other special characters that we should test against?

Reply:
Good point. I was wondering as well if there are any other special characters that would be generated by Tableau. But  I think to support the space in alias is enough based on the other chars (eg. "_" etc.) that have been supported to pass the parse so far.

Reply:
SG